packages/gatsby/package.json

```diff
@@ -102,7 +102,7 @@
     "raw-loader": "^0.5.1",
     "react-dev-utils": "^4.2.1",
     "react-error-overlay": "^3.0.0",
-    "react-hot-loader": "^4.6.0",
```

Comment:
`4.6.0` contains a small bug where anonymous arrow functions can't really be distinguished from others, e.g. for page components authored as arrow functions it was unclear which page would be mounted 😱

I've validated the fix in DSchau/gatsby-multiple-pages-issue#1